### PR TITLE
133 ensure each response is a return statement for consistency

### PR DESCRIPTION
**Pull request by 5thaks07**
complete #133

### CODE REVIEW

File: controllers/user.controller.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ export const getUserById = async (req, res) => {
         res.json(user);
```
Comment (TomStrong) [inline controllers/user.controller.js:19]: Missing one here

**Review by TomStrong — CHANGES_REQUESTED** [review]
Just missing one

File: controllers/user.controller.js (hunk 1 above)
Comment (5thaks07) [reply controllers/user.controller.js]: done

**Review by TomStrong — APPROVED** [review]
Nice one

**Review by ali1373107 — APPROVED** [review]

**Pull request merged**